Source/WebKit/Platform/cocoa/ExtensionCapabilityGrant.mm

```diff
@@ -28,21 +28,21 @@
 
 #if ENABLE(EXTENSION_CAPABILITIES)
 
+#import <BrowserEngineKit/BECapability.h>
 #import "ExtensionKitSPI.h"
 #import "Logging.h"
 #import <wtf/CrossThreadCopier.h>
 
 namespace WebKit {
 
-static void platformInvalidate(RetainPtr<_SEGrant>&& platformGrant)
+static void platformInvalidate(RetainPtr<BEProcessCapabilityGrant>&& platformGrant)
```

Comment:
Why is this taking a `RetainPtr<>&&`, we don't seem to adopt the grant or transfer ownership in any way. We can probably pass a `BEProcessCapabilityGrant *`

Reply:
Would you be OK to keep these for now, since they are virtual functions and will require further changes?

Thanks for reviewing?

Reply:
Actually, the changes aren't that many, so I'll go ahead and make that change :)

Reply:
You didn't make that change?